schedule/functional/extra_tests_gnome.yaml

```diff
@@ -0,0 +1,10 @@
+name:   extra_tests_gnome
+description:    >
+    Maintainer: asmorodskyi, okurz.
+    Extra tests which were designed to run on gnome , VNC_STALL_THRESHOLD is needed for xen svirt to don't turn off the scrreen after default 4 sec
```

Comment:
To avoid confusion, it should mention explicitly that is to test packages from following modules

- Basesystem
- Server applications
- Desktop applications

Since `>` allows multi line, VNC_STALL_THRESHOLD can be split in another line.

Reply:
modules list corrected